migrations/20200515091639-add-coloumn-User.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
careful with filenames. maintain standards, spell right. It'll get easier to track when prs have around 5 files changed at most